.github/ISSUE_TEMPLATE/Feature_Request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: 🚀 Feature Request
+about: I have a suggestion (and might want to implement myself 🙂)!
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
I'd like to ask people to read these points: https://github.com/denysdovhan/spaceship-prompt/blob/master/CONTRIBUTING.md#sections